pyproject.toml

```diff
@@ -52,7 +52,7 @@ bt = "^0.2.9"
 vaderSentiment = "3.3.2"
 GitPython = "^3.1.17"
 fred = "^3.1"
-financedatabase = "^2.0.6"
+financedatabase = {version = "2.0.7", extras = ["all"]}
```

Comment:
this wont install on poetry install.  To do this, you want to run `poetry add financedatatbase@latest`

Then 
```
poetry export -f requirements.txt --without-hashes --output requirements.txt
poetry export -f requirements.txt --with dev -E installer -E all --without-hashes --output requirements-full.txt
```